Assets/Tests/InputSystem/CoreTests_ProjectWideActions.cs

```diff
@@ -44,48 +95,38 @@ public void OneTimeSetUp()
         // This is for verifying the default output of templated actions from editor tools.
         if (File.Exists(ProjectWideActionsAsset.defaultAssetPath))
         {
-            if (!Directory.Exists(kAssetBackupDirectory))
-                AssetDatabase.CreateFolder("Assets", "~TestBackupFiles");
+            if (!Directory.Exists(m_AssetBackupDirectory))
+                Directory.CreateDirectory(m_AssetBackupDirectory);
             AssetDatabase.MoveAsset(oldPath: ProjectWideActionsAsset.defaultAssetPath,
-                newPath: kDefaultProjectWideAssetBackupPath);
+                newPath: s_DefaultProjectWideAssetBackupPath);
         }
 #endif // UNITY_EDITOR
     }
 
     [OneTimeTearDown]
-    public void OneTimeTearDown()
+    public void OneTimeTearDown() // TODO Remove
     {
 #if UNITY_EDITOR
         // Restore default asset if we made a backup copy of it during setup
-        if (File.Exists(kDefaultProjectWideAssetBackupPath))
+        if (File.Exists(s_DefaultProjectWideAssetBackupPath))
         {
             if (File.Exists(ProjectWideActionsAsset.defaultAssetPath))
                 AssetDatabase.DeleteAsset(ProjectWideActionsAsset.defaultAssetPath);
-            AssetDatabase.MoveAsset(oldPath: kDefaultProjectWideAssetBackupPath,
+            AssetDatabase.MoveAsset(oldPath: s_DefaultProjectWideAssetBackupPath,
                 newPath: ProjectWideActionsAsset.defaultAssetPath);
-            AssetDatabase.DeleteAsset(kAssetBackupDirectory);
+            Directory.Delete("Assets/~TestBackupFiles");
```

Comment:
Is there a reason t this using m_AssetBackupDirectory?
(and the second could append .meta)

Reply:
Fixed in latest commit